packages/web/design-system/components/input.tsx

```
@@ -27,39 +51,195 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
+    // Validation function
+    const validateInput = React.useCallback(
+      (inputValue: string) => {
+        for (const rule of validation) {
+          switch (rule.type) {
+            case 'required':
+              if (!inputValue.trim()) {
+                return rule.message
+              }
+              break
+            case 'email':
+              const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)